cypress/e2e/sidebarNav.cy.ts

```diff
@@ -0,0 +1,72 @@
+describe('Environment sidebar navigation', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env().CY_EMAIL, Cypress.env().CY_PASSWORD);
+  });
+
+  it('Overview/Deployments/Backups/Tasks/Vars/Problems/Facts/Insights', () => {
+    const suffix = '/projects/drupal-example/drupal-example-develop';
+    cy.visit(`${Cypress.env().CY_URL}${suffix}`);
+
+    context('From /Overview to /Deployments', () => {
+      cy.wait(3500);
+
+      cy.get('.deployments').click();
+
+      cy.location('pathname').should('equal', `${suffix}/deployments`);
+    });
+
+    cy.wait(3500);
+
+    context('From /deployments to /backups', () => {
+      cy.get('.backups').click();
+
+      cy.location('pathname').should('equal', `${suffix}/backups`);
+    });
+
+    cy.wait(3500);
+
+    context('From /backups to /tasks', () => {
+      cy.get('.tasks').click();
+
+      cy.location('pathname').should('equal', `${suffix}/tasks`);
+    });
+
+    cy.wait(3500);
+
+    context('From /tasks to /variables', () => {
+      cy.get('.environmentVariables').click();
+
+      cy.location('pathname').should('equal', `${suffix}/environment-variables`);
+    });
+
+    cy.wait(3500);
+
+    context('From /variables to /problems', () => {
+      cy.get('.problems').click();
+
+      cy.location('pathname').should('equal', `${suffix}/problems`);
+    });
+
+    cy.wait(3500);
+
+    context('From /problems to /facts', () => {
```

Comment:
Since we're going to use a specific project in the ci as well, I'm thinking problemsUI and factUI should be enabled ( for further specific tests on these pages )